demonstrations_v2/tutorial_dqi/requirements.in

```
@@ -0,0 +1,2 @@
+# Pinned as pennylane 0.42.2 requires this version
+autoray==0.6.11
```

@daniela-angulo

This pinned version of `autoray` should be fixed when the latest patch version has gone live, we should confirm before going to publish next week, but for now I will add the same range that Core is using in Pennylane.

CC: @andrijapau for transparency 

```suggestion
autoray==0.6.11,<0.8
```

---

@daniela-angulo :)

---

I asked, but still don't know. I'll let you know when I so, maybe here or over slack.